config/routes.rb

```diff
@@ -1,6 +1,4 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root "store#homepage", as: 'store_homepage', via: :all
```

Comment:
what does via: :all do?

Reply:
via: :all specifies that the root should respond to all HTTP verbs. But since here we only want to do GET requests so we can remove it